Two small tweaks

jason0597 opened the pull request:
1. Small reminder in homebrew-launcher-(soundhax) to tell users to download the `.3dsx` of safehax, to reduce the "do i download smdh or 3dsx?" questions

1.  Tell users they need at least 1.3GB (to cover both O3DS _and_ N3DS) for the NAND backup
To prevent incidents like this one:

![img](https://image.prntscr.com/image/kY4MTjWFQ-aghbEkv-UU9Q.png)

Plailect commented:
Where do you get your info on the NAND size from? Mine are smaller (1.30GB)

Plailect commented on _pages/en_US/homebrew-launcher-(soundhax).txt:
```diff
@@ -25,7 +25,7 @@ This means that using a Cart Update from a version containing an older Nintendo
 * The latest release of [SafeB9SInstaller](https://github.com/d0k3/SafeB9SInstaller/releases/latest)
 * The latest release of [boot9strap](https://github.com/SciresM/boot9strap/releases/latest) *(standard boot9strap; not the `devkit` file, not the `ntr` file)*
-* The latest release of [safehax](https://github.com/TiniVi/safehax/releases/latest)
+* The latest release of [safehax](https://github.com/TiniVi/safehax/releases/latest) *(the `.3dsx`; not the `.smdh`)*
```
should just be

    *(the `.3dsx` file)*

jason0597 commented:
I asked people on the discord channel with N3DSes, and they told me that the trimmed NAND backup is about 1.2GB. I added an additional 100MB in case they have files in their NAND (Like TWL images/games)

As for safehax, I changed it
